services/email_capture/email_capture.rst

```diff
@@ -1,50 +1,50 @@
-.. _doc_direct_deliver:
+.. _doc_email_capture:
 
 .. _Mailsac website: https://mailsac.com
 .. _Mailsac API: https://mailsac.com/api
 
-Direct Delivery
+Email Capture
 ===============
 
-Direct Delivery is the service that allows Mailsac to act as a fake SMTP
-server. Email messages sent using Direct Delivery will not be delivered to
+Email Capture is the service that allows Mailsac to act as a fake SMTP
```

Comment:
what about "fake outbound SMTP sender server" ? or something indicating it is a fake sender?

Reply:
IMO the paragraph already states it is a fake server and explains concisely in non-technical terms that emails sent using the email capture service will not be delivered outside of mailsac.